app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.kt

```diff
@@ -590,11 +530,6 @@ class CreateEventActivity : ActivityForAccessingTumOnline<CreateEventResponse>(R
     }
 
     private fun showErrorDialog(message: String) {
-        ThemedAlertDialogBuilder(this)
-            .setTitle(R.string.error)
-            .setMessage(message)
-            .setIcon(R.drawable.ic_error_outline)
-            .setPositiveButton(R.string.ok, null)
-            .show()
+        ThemedAlertDialogBuilder(this).setTitle(R.string.error).setMessage(message).setIcon(R.drawable.ic_error_outline).setPositiveButton(R.string.ok, null).show()
```

Comment:
For builders, I find that separating them on different lines is more readable. 
Is this changed mandated by another tool?

Reply:
I've split it back to multiple lines, my android studio configuration used too many spaces in the standard configuration which led to the faulty indentation